Practical-2

gauravMori158 opened the pull request:
Inside Feature Branch

gauravMori158 committed:
Practical-2

Inside Feature Branch

sagar-vaja-simform commented on Practical2-Account/Customer_Account.cs:
```diff
@@ -0,0 +1,42 @@
+        {
+            System.Console.WriteLine("Output");
+            Console.WriteLine("Bank Name : " + bank_name);
+            Console.WriteLine("Customer Name : " + customer_name);
```
return string in formatted way and print it using console.writeline in program

gauravMori158 committed:
Update Customer_Account.cs

Updated PrintInfo Method That returns well formatted string that contains user Information.
